src/types/requests.js

```diff
@@ -164,3 +164,11 @@ export type SchemesRequestsStatuses = {|
     getSchemeTeams: RequestStatusType,
     getSchemeChannels: RequestStatusType
 |};
+
+export type GroupsRequestsStatuses = {|
+    linkGroupSyncable: RequestStatusType,
+    unlinkGroupSyncable: RequestStatusType,
+    getGroupSyncables: RequestStatusType,
+    getGroupMembers: RequestStatusType,
+    getGroup: RequestStatusType,
+|};
```

Comment:
Could you add the new channel and admin requests here too?

Reply:
Fixed